<launch>

    <node pkg="image_publisher" type="image_publisher" name="image_publisher" args="$(find tl_tvm)/images/nishishinjuku_n1_2020-06-03-13-48-46000020-0.jpg">
        <param name="frame_id" value="my_camera" />
        <param name="publish_rate" value="1" />
    </node>


    <node name="tlc" pkg="nodelet" type="nodelet" args="standalone tl_tvm/TrafficLightClassifierNodelet" output="screen">
        <remap from="~input" to="image_publisher/image_raw" />
        <param name="model_file_path" value="$(find tl_tvm)/data/mobilenetv2.so" />
        <param name="model_json_path" value="$(find tl_tvm)/data/model_graph.json" />
        <param name="model_params_path" value="$(find tl_tvm)/data/model_graph.params" />
    </node>

</launch>